torchvision/models/detection/backbone_utils.py

```diff
@@ -122,3 +122,42 @@ def _validate_resnet_trainable_layers(pretrained, trainable_backbone_layers):
         trainable_backbone_layers = 3
     assert trainable_backbone_layers <= 5 and trainable_backbone_layers >= 0
     return trainable_backbone_layers
+
+
+def mobilenet_fpn_backbone(
+    backbone_name,
+    pretrained,
+    norm_layer=misc_nn_ops.FrozenBatchNorm2d,
+    trainable_layers=2,
+    returned_layers=None,
+    extra_blocks=None
+):
+    backbone = mobilenet.__dict__[backbone_name](
+        pretrained=pretrained,
+        norm_layer=norm_layer).features
+
+    # Gather the indeces of blocks which are strided. These are the locations of C1, ..., Cn-1 blocks.
+    # The first and last blocks are always included because they are the C0 (conv1) and Cn.
+    stage_indeces = [0] + [i for i, b in enumerate(backbone) if getattr(b, "is_strided", False)] + [len(backbone) - 1]
```

Comment:
Instead of adding meta-data with the location on the blocks that downsample, I get it by checking a new attribute called `is_strided`. This attribute is added in both the MobileNetV2 and V3 residual blocks and indicates if the specific block downsamples. This is typically the location of C1...Cn-1 blocks.

Note that blocks at first and last position of the features block are always included.